src/receivers/http-utils.ts

```diff
@@ -1,5 +1,6 @@
 import { IncomingMessage } from 'http';
 
+// Deprecated: this function will be removed in the near future
 export function extractRetryNum(req: IncomingMessage): number | undefined {
```

Comment:
I didn't remove these methods this time. These methods are no longer used. Also, they are not exported from `src/index.ts`. Thus, deleting this whole file should be safe enough.

Reply:
Great idea. Will add some logs